src/CreateUsernamePage.jsx

```diff
@@ -22,18 +25,14 @@ const CreateUsernamePage = () => {
     }
 
     try {
-      const response = await axios.post("http://localhost:3001/register", {
-        name: email,
+      await addDoc(collectionReference, {
+        email: email
       });
-
-      if (response.data.exists) {
-        setErrorMessage(response.data.message);
-      } else {
-        navigate("/");
-      }
+      setErrorMessage("Email Registered Succesfully");
+      setTimeout(()=>navigate("/"),1000 )
     } catch (error) {
       console.error("Error registering user:", error);
-      setErrorMessage("An error occurred while registering.");
+      setErrorMessage("Error lil bro");
```

Comment:
No, lets not send troll messages. change it back to "An error occurred while registering".